config/master/manifests.jsonnet

```diff
@@ -197,15 +197,15 @@ local awsnode = {
                 {mountPath: "/host/etc/cni/net.d", name: "cni-net-dir"},
                 {mountPath: "/host/var/log/aws-routed-eni", name: "log-dir"},
                 {mountPath: "/var/run/aws-node", name: "run-dir"},
-                {mountPath: "/var/run/dockershim.sock", name: "dockershim"},
+                {mountPath: "/var/run/cri.sock", name: "cri-sock"},
               ],
             },
           },
           containers: objectValues(self.containers_),
           volumes: [
             {name: "cni-bin-dir", hostPath: {path: "/opt/cni/bin"}},
             {name: "cni-net-dir", hostPath: {path: "/etc/cni/net.d"}},
-            {name: "dockershim", hostPath: {path: "/var/run/dockershim.sock"}},
+            {name: "cri-sock", hostPath: {path: "/var/run/dockershim.sock"}},
```

Comment:
Should we just keep it as dockershim and update the readme with clear comments or have another sample config with cri sock?

Reply:
The point here is that it's not docker specific, it's "any CRI socket" that we mount inside the pod. That's why we want to rename it.

Reply:
Considering that Kubernetes is deprecating Docker, should this be merged? Would make make it easier to understand what changes are require for compatibility with containerd or cri-o.

It may be a good idea to also remove mentions to `dockerSocketPath` at same time:
https://github.com/aws/amazon-vpc-cni-k8s/blob/1c1d4b91e8d21d979cad494fe8228c464d1d4d3b/pkg/cri/cri.go#L54-L57